spec/requests/v1/pacbio/runs_spec.rb

```diff
@@ -758,14 +761,21 @@
                       include_fivemc_calls_in_cpg_motifs: 'Yes',
                       ccs_analysis_output_include_kinetics_information: 'Yes',
                       demultiplex_barcodes: 'In SMRT Link',
-                      pool_ids: [pool1.id, pool1.id] }
+                      used_aliquots_attributes: [{ source_id: pool1.id, source_type: 'Pacbio::Pool', volume: 10, concentration: 20, aliquot_type: :derived, template_prep_kit_box_barcode: '033000000000000000000' },
+                                                 { source_id: pool2.id, source_type: 'Pacbio::Pool', volume: 10, concentration: 20, aliquot_type: :derived, template_prep_kit_box_barcode: '033000000000000000000' }] }
                   ]
                 }]
               }
             }
           }.to_json
         end
 
+        before do
+          if pool2.used_aliquots.any? && pool1.used_aliquots.any?
```

Comment:
Is this `if` needed? Conditional checks for setting up test data can lead to flakey tests. If this is needed for a specific test then include this in that test.